pkg/server/registry/apigroups/acorn/events/testdata/query/TestQuery/full/expected.golden

```diff
@@ -0,0 +1,5 @@
+`0a6986e5dc34a7f211db5f8881d9160e AppCreate           "2023-06-08T13:32:36.345221311-04:00"
```

Comment:
The timezone is causing a diff when running these tests in CI. I think this is a side-effect of the way I'm using jsonpath.